runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -237,5 +268,17 @@ private void renderZalcanoAOE(Graphics2D graphics, int polySize, String text, Co
 		}
 	}
 
+	private void renderPlayerCount(Graphics2D graphics)
```

Comment:
It would be better to move this and the minimap overlay to a seperate class, so you can use `OverlayLayer.ABOVE_WIDGETS` and layout anchors

Reply:
hm separate ovelay then. It shall be called minioverlay!